fight.py

```diff
@@ -1,5 +1,5 @@
 import random
-
+from inventory import Inventory
```

Comment:
I like that you grouped these functions in their own file! Even if it's not a class, it makes the entire program readable and better structured.

Reply:
For future projects, a lot of times people will put general helper functions in whats called a "utils" file, similar to what you did here!